app/models/user.rb

```diff
@@ -643,11 +643,6 @@ def bump_last_seen_reviewable!
     if max_reviewable_id
       update!(last_seen_reviewable_id: max_reviewable_id)
       publish_reviewable_counts(unseen_reviewable_count: self.unseen_reviewable_count)
-      MessageBus.publish(
```

Comment:
We could probably add an example to the `User` specs to check we’re publishing the proper event. If we had that when you did your refactor, it would probably have caught that the event was sent twice. Wdyt?